tasks/test_results_processor.py

```diff
@@ -337,7 +337,9 @@ def create_daily_total():
 
         # Upsert Tests
         if len(test_data) > 0:
-            test_insert = insert(Test.__table__).values(list(test_data.values()))
+            test_insert = insert(Test.__table__).values(
+                sorted(list(test_data.values()), key=lambda x: str(x["flags_hash"]))
```

Comment:
```suggestion
                sorted(test_data.values(), key=lambda x: str(x["flags_hash"]))
```

`sorted` takes an iterator.